pkg/config/parser.go

```diff
@@ -53,9 +53,10 @@ type LoaderConfiguration struct {
 	MetricScrapingPeriodSeconds int    `json:"MetricScrapingPeriodSeconds"`
 	AutoscalingMetric           string `json:"AutoscalingMetric"`
 
-	GRPCConnectionTimeoutSeconds int  `json:"GRPCConnectionTimeoutSeconds"`
-	GRPCFunctionTimeoutSeconds   int  `json:"GRPCFunctionTimeoutSeconds"`
-	DAGMode                      bool `json:"DAGMode"`
+	GRPCConnectionTimeoutSeconds int    `json:"GRPCConnectionTimeoutSeconds"`
+	GRPCFunctionTimeoutSeconds   int    `json:"GRPCFunctionTimeoutSeconds"`
+	DAGMode                      bool   `json:"DAGMode"`
```

Comment:
Lines 56-58 seem to have only added spaces. I suggest keeping them as they were originally.

Reply:
That is gofmt style to align with the other lines. This should be this way